api_tests/requests/views/test_node_request_institutional_access_logging.py

```diff
@@ -0,0 +1,97 @@
+import pytest
```

Comment:
New tests only have add behavior, remove curator is not done via the NodeRequetAction endpoint so those curator removal tests are in the project view contributor management test files.